install/centos/README.md

```diff
@@ -89,52 +90,45 @@ Now install the `epel-release-6-8.noarch` package, which will enable EPEL reposi
+    wget -O /etc/pki/rpm-gpg/RPM-GPG-KEY-remi http://rpms.famillecollet.com/RPM-GPG-KEY-remi
+    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-remi
 
 Verify that the key got installed successfully:
 
     rpm -qa gpg*
-    gpg-pubkey-41a40948-4ce19266
+    gpg-pubkey-00f97f56-467e318a
+
+Now install the `remi-release-6` package, which will enable remi-safe repository on your system:
 
-Verify that the EPEL and PUIAS Computational repositories are enabled as shown below:
+    rpm -Uvh http://rpms.famillecollet.com/enterprise/remi-release-6.rpm
+
+Verify that the EPEL and remi-safe repositories are enabled as shown below:
 
     yum repolist
 
-    repo id                 repo name                                                status
-    PUIAS_6_computational   PUIAS computational Base 6 - x86_64                      2,018
-    base                    CentOS-6 - Base                                          4,802
-    epel                    Extra Packages for Enterprise Linux 6 - x86_64           7,879
-    extras                  CentOS-6 - Extras                                           12
-    updates                 CentOS-6 - Updates                                         814
-    repolist: 15,525
+    repo id        repo name                                                       status
+    base           CentOS-6 - Base                                                  6696
+    epel           Extra Packages for Enterprise Linux 6 - x86_64                  12125
+    extras         CentOS-6 - Extras                                                  61
+    remi-safe      Safe Remi's RPM repository for Enterprise Linux 6 - x86_64        827
+    updates        CentOS-6 - Updates                                                137
+    repolist: 19846
 
 If you can't see them listed, use the folowing command (from `yum-utils` package) to enable them:
 
-    yum-config-manager --enable epel --enable PUIAS_6_computational
+    yum-config-manager --enable epel --enable remi-safe
 
 ### Install the required tools for GitLab
 
     yum -y update
     yum -y groupinstall 'Development Tools'
-    yum -y install readline readline-devel ncurses-devel gdbm-devel glibc-devel tcl-devel openssl-devel curl-devel expat-devel db4-devel byacc sqlite-devel libyaml libyaml-devel libffi libffi-devel libxml2 libxml2-devel libxslt libxslt-devel libicu libicu-devel system-config-firewall-tui redis sudo wget crontabs logwatch logrotate perl-Time-HiRes git cmake libcom_err-devel.i686 libcom_err-devel.x86_64 nodejs
+    yum -y install readline readline-devel ncurses-devel gdbm-devel glibc-devel tcl-devel openssl-devel curl-devel expat-devel db4-devel byacc sqlite-devel libyaml libyaml-devel libffi libffi-devel libxml2 libxml2-devel libxslt libxslt-devel libicu libicu-devel system-config-firewall-tui redis sudo wget crontabs logwatch logrotate perl-Time-HiRes git cmake libcom_err-devel.i686 libcom_err-devel.x86_64 nodejs python-docutils
```

Comment:
I used [Ubuntu guide](https://github.com/gitlabhq/gitlabhq/blob/master/doc/install/installation.md) as a base for updating this one and docutils and rst are not marked there as optional.